contracts/strategy/rysk/RyskStrategy.sol

```diff
@@ -377,4 +346,60 @@ contract RyskStrategy is BaseStrategy {
     function _getSharesBalance() internal view returns (uint256) {
         return ryskLqPool.balanceOf(address(this));
     }
+
+    /**
+     * Gets the amount of underlying from the deposit receipt which includes the pending deposit amount and unreddemed shares.
+     *
+     * @return amount of underlying form the deposit receipt
+     */
+    function _getAmountFromDepositReceipt() internal view returns (uint256) {
+        IRyskLiquidityPool.DepositReceipt
+            memory depositReceipt = _getDepositReceipt();
```

Comment:
don't you want to maybe check if there's anything in the deposit receipt before you run the rest of the code?

Reply:
that could save some gas, added the [check](https://github.com/lindy-labs/sc_solidity-contracts/pull/216/commits/070e898758ce4800b05c9f4acdd08fa37cd1ebd2) for amount and unredeemed shares